lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -259,6 +261,137 @@ class ExportCommand < HammerCLIKatello::SingleResourceCommand
       end
     end
 
+    class ExportCommand < HammerCLIKatello::SingleResourceCommand
```

Comment:
This should probably be:

```rb
class ExportCommand < HammerCLI::AbstractCommand
  include HammerCLI::Messages
  ...
end
```

Using any command classes that have the `HammerCLI::Apipie::Command` ancestor will give you apipie-knowledgeable commands, and this command doesn't associate with an apipie controller action definition.

Reply:
Making that change fails earlier:

```
TypeError (no implicit conversion from nil to integer):
    /root/hammer-cli-katello/lib/hammer_cli_katello/content_view_version.rb:303:in `insert'
    /root/hammer-cli-katello/lib/hammer_cli_katello/content_view_version.rb:303:in `option_sources'
```

Any idea what is causing that or how to fix it based on the class change?